initrd-builder/README.md

```diff
@@ -0,0 +1,25 @@
+* [Creating a guest OS initrd image](#creating-a-guest-os-initrd-image)
+* [Further information](#further-information)
+
+# Kata Containers initrd image generation
+
+A Kata Containers initrd image is generated using the `initrd_builder.sh` script.
+This uses a rootfs directory created by the `rootfs-builder/rootfs.sh` script.
+
+## Creating a guest OS initrd image
+
+To create a guest OS initrd image run:
+
+```
+$ sudo ./initrd_builder.sh path/to/rootfs
+```
+
+Where `path/to/rootfs` is the directory populated by `rootfs.sh`.
+
+## Further information
+
+For more information about usage, run:
```

Comment:
Unspecific, consider changing to:
For more information on how to use the `initrd_builder.sh` script, run: